packages/IMAPClient-Core.package/ICEmail.class/instance/senderName.st

```diff
@@ -3,4 +3,4 @@ senderName
 
 	| fromCollection |
 	fromCollection := self from findBetweenSubStrs: self class senderAddressDelimeters.
-	^ (fromCollection at: 1 ifAbsent: [ '' ])
\ No newline at end of file
+	^ ((fromCollection at: 1 ifAbsent: [ '' ]) subStrings: '"') at: 1
```

Comment:
Changing the default value to an empty string is a different user story. Please remove this change

Reply:
You maybe misunderstood what this does. This removes the quotation marks (which are sometimes included in the sender name) to make a consistent sorting order possible.